zebra-state/src/service/finalized_state.rs

```diff
@@ -390,6 +396,15 @@ impl FinalizedState {
                 batch.zs_insert(history_tree_cf, height, history_tree);
             }
 
+            let current_pool = self.current_value_pool().constrain::<NegativeAllowed>()?;
+            batch.zs_insert(
+                tip_chain_value_pool,
+                height,
```

Comment:
Can we just store the tip chain value pool, rather than storing the chain value pool for every height?
```suggestion
                (),
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2599/commits/26b40d913974e1f4a8f74b53a9330127edb84567